editor/plugins/animation_state_machine_editor.h

```diff
@@ -41,11 +41,17 @@ class ConfirmationDialog;
 class EditorFileDialog;
 class OptionButton;
 class PanelContainer;
+class HSplitContainer;
+class MarginContainer;
+class HSeparator;
+class CheckBox;
+class AnimationNodeAnimation;
 
 class AnimationNodeStateMachineEditor : public AnimationTreeNodeEditorPlugin {
 	GDCLASS(AnimationNodeStateMachineEditor, AnimationTreeNodeEditorPlugin);
 
 	Ref<AnimationNodeStateMachine> state_machine;
+	// Ref<AnimationNodeAnimation> node;
```

Comment:
```suggestion
```

Reply:
12 also used a lot in animation_node_state_machine, would it be good to add `STATE_MACHINE_GROUPS_LIMIT` in there too good?

Reply:
If it is defined as static, you should be able to get by `AnimationNodeStateMachineEditor::STATE_MACHINE_GROUPS_LIMIT`.

Reply:
Or, it is better to define it in the TOOLS_ENABLED block of state_machine.h, since it does not require more includes.

Reply:
uint64_t gives warning for loops when compared with int's. 

What's the right way to get STATE_MACHINE_GROUPS_LIMIT?

Best one so far was making it public and using it as AnimationNodeStateMachine::STATE_MACHINE_GROUPS_LIMIT 
```C++
class AnimationNodeStateMachine : public AnimationRootNode {
	GDCLASS(AnimationNodeStateMachine, AnimationRootNode);

public:

#ifdef TOOLS_ENABLED
	static constexpr int STATE_MACHINE_GROUPS_LIMIT = 12;
#endif //TOOLS_ENABLED
```

Reply:
For now I made var static constexpr int and public and used it as AnimationNodeStateMachine::STATE_MACHINE_GROUPS_LIMIT without defining anything on animation_state_machine_editor.h. I am uploading it as this but I will keep this as unresolved and wait for your suggestion.

Reply:
It's fine.